common/config/presets/windows.toml

```diff
@@ -400,3 +400,7 @@ monerod_password=""
 
 ## Will start mining only when base node has reported that it is bootstrapped
 #mine_on_tip_only=true
+
+## Will check tip with node every N seconds and restart mining if height already taken and
+## option `mine_on_tip_only` is set to true
+#validate_tip_timeout_sec=30
```

Comment:
Please add all `[mining_node]` config setting to `tari_config_example.toml`, with the minimal settings not commented out.

I know it is a bit confusing, and on our TODO list to fix/consolidate the config files. This is the current state:
- `tari_sample.toml` has minimal settings for all Tari executables, but is not really used in our official releases
- `windows.toml` is used to create the Windows installer, and apart from the path separators identical to `tari_config_example.toml`
- `tari_config_example.toml` is used to create the OSX and Ubuntu installation tarbals, and apart from the path separators identical to `windows.toml`

Reply:
Cool, that's the file I was missing, will add there thanks